frontend/src/types.ts

```diff
@@ -2177,6 +2177,7 @@ export interface RatingSurveyQuestion extends SurveyQuestionBase {
 export interface MultipleSurveyQuestion extends SurveyQuestionBase {
     type: SurveyQuestionType.SingleChoice | SurveyQuestionType.MultipleChoice
     choices: string[]
+    has_open_choice?: boolean
```

Comment:
maybe camel case since it's only ever used in the frontend? Like `lowerBoundLabel` in rating survey question

```suggestion
    hasOpenChoice?: boolean
```